test/astar.cc

```diff
@@ -438,24 +438,36 @@ void DoConfig() {
   write_config(config_file);
 }
 
-std::string ways_file = "test_ways_whitelion.bin";
-std::string way_nodes_file = "test_way_nodes_whitelion.bin";
-std::string access_file = "test_access_whitelion.bin";
-std::string from_restriction_file = "test_from_complex_restrictions_whitelion.bin";
-std::string to_restriction_file = "test_to_complex_restrictions_whitelion.bin";
-std::string bss_file = "test_bss_nodes_whitelion.bin";
-
 boost::property_tree::ptree get_conf(const char* tiles) {
   std::stringstream ss;
   ss << R"({
-      "mjolnir":{"tile_dir":"test/data/)"
-     << tiles << R"(", "concurrency": 1},
+      "mjolnir":{
+        "tile_dir":"test/data/)"
+     << tiles << R"(",
+        "concurrency": 1
+      },
       "loki":{
         "actions":["route"],
         "logging":{"long_request": 100},
-        "service_defaults":{"minimum_reachability": 2,"radius": 10,"search_cutoff": 35000, "node_snap_tolerance": 5, "street_side_tolerance": 5, "heading_tolerance": 60}
+        "service_defaults":{
+          "minimum_reachability": 50,
+          "radius": 0,
```

Comment:
To actually achieve the failure conditions, radius had to be something less than the older value.